setup.py

```diff
@@ -31,7 +31,7 @@
         scripts/generate_prsn
         scripts/split_merged_climos
         scripts/update_metadata
-	    scripts/decompose_flow_vectors
+		scripts/decompose_flow_vectors
```

Comment:
You're mixing tabs and spaces. I think you need some work on configuring your editor.